basics/datagrid/src/index.ts

```diff
@@ -26,34 +22,34 @@ const extension: JupyterFrontEndPlugin<void> = {
     mainMenu: IMainMenu
   ) => {
     const { commands, shell } = app;
-    let command = 'ex4:datagrid';
-    let category = 'Tutorial';
+    let command = 'examples:datagrid';
+    let category = 'Example';
     commands.addCommand(command, {
-      label: 'Ex4 datagrid',
+      label: 'Datagrid example',
       caption: 'Open a datagrid panel',
-      execute: args => {
-        const widget = new TutorialView();
+      execute: () => {
+        const widget = new ExampleView();
         shell.add(widget, 'main');
       }
     });
     palette.addItem({ command, category });
 
-    let tutorialMenu: Menu = new Menu({ commands });
+    let exampleMenu: Menu = new Menu({ commands });
```

Comment:
Maybe we can make this variable `const`?

Reply:
Done, I also remove the typing as Typescript will solve it easily.